pyaedt/edb_core/EDB_Data.py

```diff
@@ -1157,52 +1318,56 @@ def rotation(self):
 
     @property
     def placement_layer(self):
-        """
+        """Retrieve the placement layer.
 
         Returns
         -------
         str
+            Placement layer.
         """
         return self.pin.GetGroup().GetPlacementLayer().GetName()
 
     @property
     def lower_elevation(self):
-        """
+        """Retrieve the lower elevation of the placement layer.
 
         Returns
         -------
         float
+            Lower elavation of the placement layer.
         """
         return self.pin.GetGroup().GetPlacementLayer().GetLowerElevation()
 
     @property
     def upper_elevation(self):
-        """
+        """Retrieve the upper elevation of the placement layer.
 
         Returns
         -------
         float
+           Upper elevation of the placement layer.
         """
         return self.pin.GetGroup().GetPlacementLayer().GetUpperElevation()
 
     @property
     def top_bottom_association(self):
-        """
+        """Retrieve the top/bottom association of the placement layer.
 
         Returns
         -------
         int
+            Top/bottom association of the placement layer.
             0 Top associated.
             1 No association.
             2 Bottom associated.
             4 Number of top/bottom association type.
             -1 Undefined.
```

Comment:
```suggestion

            * 0 - Top associated.
            * 1 - No association.
            * 2 - Bottom associated.
            * 4 - Number of top/bottom association type.
            * -1 - Undefined.
```

When creating bulleted lists in rst, make sure there's a space between the line above it or else it won't render correctly.  Additionally, each line needs to have a `*` before it.